gutils/collections.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 Google LLC
```

Comment:
Please use the copyright notice from here: https://github.com/p4lang/p4-constraints/blob/master/p4_constraints/ast.h#L1-L15

Reply:
Discussed offline. Keeping current one as in rest of folder.